src/site/_data/contributors.js

```diff
@@ -585,7 +585,7 @@ const contributors = {
     github: 'jpmedley',
     twitter: 'medleyjp',
     description: {
-      en: "If an API's not documented it doesn't exist",
+      en: `If an API's not documented it doesn't exist`,
```

Comment:
I think using double quotes for strings that contain a single quote would be ok... does the linter grump about that? is that why they all needed to be switched to backticks?